src/tabs/failsafe.html

```diff
@@ -139,6 +139,12 @@
                                     </label>
                                     <div class="helpicon cf_tip" i18n_title="failsafeGpsRescueItemMinDthHelp"></div>
                                 </div>
+                                <div class="number">
+                                    <label> <input type="number" name="gps_rescue_initial_climb" min="5" max="1000" /> <span
+                                        i18n="failsafeGpsRescueInitialClimb"></span>
+                                    </label>
+                                    <div class="helpicon cf_tip" i18n_title="failsafeGpsRescueInitialClimbHelp"></div>
+                                </div>
                                 <div class="number">
                                     <label> <input type="number" name="gps_rescue_ground_speed" min="3.00" max="30.00" step="0.01"/> <span
                                         i18n="failsafeGpsRescueItemGroundSpeed"></span>
```

Comment:
Please change step here to 0.1, and only once decimal place for min and max, eg:
`min="3.0" max="30.0" step="0.1"`